### PR TITLE
Remove Wiki Sidebar HTML, LESS, JS

### LINKED ISSUE #9807: Remove wiki sidebar
Since we are moving to the forum, we can remove the wiki sidebar javascript, HTML and less.

### PR DESCRIPTION
**Pull request by dhcodes**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [x] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9807

### CODE REVIEW

**Comment by raisedadead** [comment]
Yikes, @dhcodes we have commits that have got in between yours. 
Can you try rebasing this of staging and then a squash?

FIRST take a back up of your changes (physically copy the files somewhere safe)

```
git checkout fix/remove-wiki-sidebar
git pull --rebase upstream staging
git log -n 10 --oneline
```

This will give you just top 10 commits 
Then you can count and squash whatever is yours.

```
git rebase -i HEAD ~ 3
git push origin fix/remove-wiki-sidebar --force
```

If the above fails or you get stuck. Below is a sure shot way.

1. Checkout staging `git checkout staging`
2. Delete `fix/remove-wiki-sidebar` by doing `git branch -D fix/remove-wiki-sidebar`
3. Create the branch again `git checkout -b fix/remove-wiki-sidebar`
4. Copy the files you saved back
5. Add them, commit them
6. Force push with `git push origin fix/remove-wiki-sidebar --force`

File: server/views/partials/footer.jade [hunk 1]
```diff
@@ -5,9 +5,5 @@ aside.map-aside.is-collapsed
   .map-aside-action-bar
       a.map-aside-action-item.map-aside-action-pop-out(href='/map' target='_blank' aria-label='open map in new tab')
       button.map-aside-action-item.map-aside-action-collapse(aria-label='close map aside')
-aside.wiki-aside.is-collapsed
-  .wiki-aside-action-bar.wiki-fixed-header
       .chat-embed-main-title
```
Comment (atjonathan) [inline server/views/partials/footer.jade]: This should be removed.

Comment (dhcodes) [inline server/views/partials/footer.jade]: Good catch. Thanks

**Comment by dhcodes** [comment]
Thanks @raisedadead for taking the time to write that up. I was sort of at a loss. I was planning to come back to it this morning anyway but forgot to close it for the time being. My Git skills are apparently not great.

**Comment by raisedadead** [comment]
@dhcodes we have all gone thru this, git is weird at start but once you understand it, it's the awesomest SCM/VCS tool ever built. Good luck and happy fixing!

**Comment by raisedadead** [comment]
@dhcodes thanks for contributing.

**Comment by dhcodes** [comment]
Thanks again for all the help @raisedadead

**Pull request merged**